src/common/Surface.js

```diff
@@ -55,6 +56,8 @@
     //Surface.prototype.publish("menu");
     Surface.prototype.publish("content", null, "widget", "Content",null,{tags:['Private']});
 
+    Surface.prototype.publish("buttonAnnotations", [], "array", "Button Array",null,{tags:['Private']});
```

Comment:
We should start using double quotes consistently (I suspect I will enable a lint warning for this at some point)